docs/index.md

```diff
@@ -1,61 +1,81 @@
-# Welcome to whisk's documentation!
+# whisk documentation
 
-**[whisk](https://github.com/whisk-ml/whisk) is a ML project framework that make makes collaboration, reproducibility, and deployment "just work".**
+__whisk is an open-source ML project framework that make makes collaboration, reproducibility, and deployment "just work".__ It combines a suite of lightweight tools with a logical and flexible project structure. Release your model to the world without a software engineer.
 
-Tying together the tools required to release a machine learning model can be daunting. Whisk makes building and releasing ML models easy and fun. Whisk creates a logical and flexible project structure for ML with reproducible results and lets you release your model to the world without becoming a software engineer.
-
-Whisk doesn't lock you into a particular ML framework or require you to learn yet another ML packaging API. Instead, it leverages the magic of Python's ecosystem that's available to projects structured in a Pythonic-way. Whisk does the structuring while you focus on the data science.
+Whisk doesn't lock you into a particular ML framework or require you to learn yet another ML packaging API. Instead, it lets you leverage the large Python ecosystem by structuring your ML project in a [Pythonic-way](
+https://docs.python-guide.org/writing/structure/). Whisk does the structuring while you focus on the data science.
 
 Read more about our [beliefs](#beliefs).
 
-## Quickstart
+## A quick tour of whisk
+
+The best way to experience whisk is by creating your first project. The short self-guided tour below shows the power of whisk.
 
-_Replace `demo` with the name of your ML project in the examples below._
+_We use __demo__ as the project name in the examples below. If you use a different project name, be sure to replace __demo__ with the name of your project._
 
-### Create a project for the model
+### Create a project
 
-Create the project with whisk:
+Start by creating a project. Begin a terminal session and run the commands below:
 
 ```
 $ pip install whisk
-$ echo "Generate the directory structure, set up a venv, initialize a Git repo, and more."
 $ whisk create demo
 $ cd demo
 $ source venv/bin/activate
 ```
 
-Take a quick tour the project you just created:
+This installs the whisk package, creates the project, and activates the project's venv.
 
-```
-$ jupyter-notebook notebooks/getting_started.ipynb
-```
+### Try the sample model
+
+The project contains a simple model stub that returns the length of each inner list within a 2D array. The model is saved to the disk using pickle. We can use the model stub to demonstrate the full capabilities of a whisk project.
```

Comment:
Curious - what would you think about changing the example model to the `iris` example or something similar? It might be more relatable to a DS who is trying to figure out how `whisk` would apply to their work. I like the simplicity of the existing example, but I worry it might be hard to translate to their project.

Reply:
I'm open to it, but I worry about:

* Do we need to add training code to this?
* If we don't add training code and serialize the model, will that model work across different Python versions?
* Do we also then to include an ML framework like scikit learn?

Reply:
As discussed in slack, we will instead make a Iris example, and link to that in the `quick tour`